setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = lc-sdk-python
-version = 0.1.10
+version = 0.1.11
```

Comment:
After decision about versioning support i would reconsider if increasing patch version is enough here.

Reply:
Skipped for now. https://github.com/livechat/lc-sdk-python/pull/60/commits/cfddc0f8c8ea174ad26361f5d3646b3e348782a7
We will establish version when releasing.